var/spack/repos/builtin/packages/athena/package.py

```diff
@@ -21,17 +21,60 @@ class Athena(AutotoolsPackage):
     # PHYSICS "packages":
     variant('problem', default='linear_wave', description='Problem generator',
             values=[
-        'blast', 'carbuncle', 'collapse3d', 'cpaw', 'cshock1d',
-        'current_sheet', 'cyladvect', 'cylblast', 'cylbphi', 'cylbr',
-        'cylcvmri', 'cyldiff', 'cylfieldloop', 'cylnewtmri', 'cylrayleigh',
-        'cylspiral', 'cylwind', 'cylwindrot', 'cylwindrotb', 'dmr', 'fft_test',
-        'field_loop', 'firehose', 'hall_drift', 'hb3', 'hgb', 'hkdisk',
-        'jeans', 'jet', 'kh', 'linear_wave', 'lw_implode', 'msa', 'noh',
-        'orszag-tang', 'par_collision', 'par_epicycle', 'par_strat2d',
-        'par_strat3d', 'pgflow', 'rotor', 'rt', 'shk_cloud', 'shkset1d',
-        'shkset2d', 'shkset3d', 'shu-osher', 'strat', 'streaming2d_multi',
-        'streaming2d_single', 'streaming3d_multi', 'streaming3d_single',
-        'twoibw'])
+                'blast',
+                'carbuncle',
+                'collapse3d',
+                'cpaw',
+                'cshock1d',
+                'current_sheet',
+                'cyladvect',
+                'cylblast',
+                'cylbphi',
+                'cylbr',
+                'cylcvmri',
+                'cyldiff',
+                'cylfieldloop',
+                'cylnewtmri',
+                'cylrayleigh',
+                'cylspiral',
+                'cylwind',
+                'cylwindrot',
+                'cylwindrotb',
+                'dmr',
+                'fft_test',
+                'field_loop',
+                'firehose',
+                'hall_drift',
+                'hb3',
+                'hgb',
+                'hkdisk',
+                'jeans',
+                'jet',
+                'kh',
+                'linear_wave',
+                'lw_implode',
+                'msa',
+                'noh',
+                'orszag-tang',
+                'par_collision',
+                'par_epicycle',
+                'par_strat2d',
+                'par_strat3d',
+                'pgflow',
+                'rotor',
+                'rt',
+                'shk_cloud',
+                'shkset1d',
+                'shkset2d',
+                'shkset3d',
+                'shu-osher',
+                'strat',
+                'streaming2d_multi',
+                'streaming2d_single',
+                'streaming3d_multi',
+                'streaming3d_single',
+                'twoibw'
+            ])
```

Comment:
Is there any other way we can have this comply with flake8 and not take up 50+ lines?

Reply:
Yeah but is it so bad?  One thing per line makes it easier to read, and minimizes diffs if you change it.  I *think* this is also what black would do to it.

Reply:
I guess it's fine... It sounds like black will make the `import_modules` lists pretty long too.